store/tikv/region_cache.go

```diff
@@ -1041,6 +1041,9 @@ func (c *RegionCache) loadRegionByID(bo *Backoffer, regionID uint64) (*Region, e
 			}
 		}
 		reg, err := c.pdClient.GetRegionByID(ctx, regionID)
+		logutil.BgLogger().Warn(fmt.Sprintf("%T", c.pdClient))
```

Comment:
?

Reply:
fixed, it should in another unpushed branch 🤣